### DISCUSSION TITLE
Change the Size of the text showing up

### DISCUSSION THREAD

**Discussion by mzfr** [adityatelange/hugo-PaperMod#219] [Q&A] [answered]
How can I change the size of the text of the blog post?

**Comment by kdkasad** [comment, marked as answer]
You can add a custom CSS file which changes the size of text. E.g.
```css
main div.post-content p {
    font-size: 14pt;
}

**Reply by mzfr** [reply]
@kdkasad do I need to add a custom CSS file under `/static` folder or under `/assets` ?

**Reply by kdkasad** [reply]
https://adityatelange.github.io/hugo-PaperMod/posts/papermod/papermod-faq/#bundling-custom-css-with-themes-assets

**Reply by mzfr** [reply]
Can you also tell me how to change the width of the centered text?

I mean when we open the post it's just in the center, which is fine but I want to change the width of that.

**Reply by kdkasad** [reply]
> Can you also tell me how to change the width of the centered text?

Do you mean the width of the text area? If so:
```css
div.post-content {
    width: 120%;
}
```

You should use a tool like *Inspect Element* to find out what you need to change. That way you don't have to keep asking here.

**Reply by mzfr** [reply]
Thanks for the help:smile:
